sky/skylet/constants.py

```diff
@@ -126,7 +126,7 @@
 # https://github.com/ray-project/ray/issues/31606
 # We use python 3.10 to be consistent with the python version of the
 # AWS's Deep Learning AMI's default conda environment.
-CONDA_INSTALLATION_COMMANDS = (
+CONDA_INSTALLATION_COMMANDS = lambda conda_auto_activate: (
```

Comment:
nit: Can we just use `.format` for this purpose? Having a lambda function in the `constants.py` is a bit weird.

Reply:
I actually tried, but `.format` does not work for those `{}` in the installation commands..

Reply:
I see, in that case, let's have a comment here for why we use `lambda` function here. : )